deploy-templates/values.yaml

```diff
@@ -8,6 +8,8 @@ global:
   platform: "kubernetes"
   # -- a cluster DNS wildcard name
   dnsWildCard:
+  # -- a cluster api gateway URL
```

Comment:
# API Gateway URL configuration for Widget Functionality
# This configuration specifies the URL of the API Gateway that is used by various widgets within the platform.
# The 'api_gateway_url' should be set to the ingress URL of the KrakenD API Gateway
# By default, this value is left empty, and widgets are disabled
# -- API Gateway URL configuration for Widget Functionality
api_gateway_url: ""